### DISCUSSION TITLE
Vercel Statsig integration. Confused about how it works.

### DISCUSSION THREAD

**Discussion by JPVRS** [vercel/examples#913] [General]
I've created this integration and a window opened to Statsig where I selected a project, chose a config name, and then got env variables for EDGE_CONFIG and EDGE_CONFIG_ITEM_KEY.

I can see the integration in our Statsig dashboard. I don't see the integration in our Vercel dashboard, either for the account or for the specific project.

When adding the integration, an edge config store was automatically created in our Vercel dashboard. When I click into that, there are no "Items".

I'm wondering how exactly the config is being stored/transmitted between Statsig and Vercel, since the edge config store contains no items and there's no integration showing up in the dashboard on Vercel's side?

Maybe it's not important to understand this, but I find it very confusing.

Edit: The integration shows up in our Vercel dashboard now, after going back to the Statsig pop-up and clicking to continue. There still aren't any "Items" in the edge config store that was created, so I guess I'm still confused as to what it's even doing?

**Comment by rezaselim381-max** [comment]
Hiya everyone
